src/controllers/handlers/utils.ts

```diff
@@ -28,6 +26,58 @@ export async function validate<T extends string>(
   return {
     identity,
     realmName,
-    sceneId
+    sceneId,
+    parcel,
+    hostname
+  }
+}
+
+export function formatUrl(url: string): string {
+  if (!url) return '/'
+  return url.endsWith('/') ? url : `${url}/`
+}
+
+export function isValidAddress(address: string): boolean {
+  if (typeof address !== 'string' || !address) return false
+  return /^0x[a-fA-F0-9]{40}$/i.test(address)
+}
+
+export function validateFilters(filters: { admin?: string }): {
+  valid: boolean
+  error?: string
+  value: { admin?: string }
+} {
+  if (filters.admin !== undefined && typeof filters.admin !== 'string') {
+    return {
+      valid: false,
+      error: 'admin must be a string',
+      value: {}
+    }
+  }
+
+  return {
+    valid: true,
+    value: {
+      admin: filters.admin ? filters.admin.toLowerCase() : undefined
+    }
+  }
+}
+
+export async function fetchDenyList(): Promise<Set<string>> {
+  try {
+    const response = await fetch('https://config.decentraland.org/denylist.json')
+    if (!response.ok) {
+      throw new Error(`Failed to fetch deny list, status: ${response.status}`)
+    }
+    const data = await response.json()
+    if (data.users && Array.isArray(data.users)) {
+      return new Set(data.users.map((user: { wallet: string }) => user.wallet.toLocaleLowerCase()))
+    } else {
+      console.warn('Deny list is missing "users" field or it is not an array.')
```

Comment:
we should not use console and use WKC logger instead

Also returning an empty Set in case of failure will provoke to by-pass this validation when the config is not available making it possible to use a blacklisted name

Reply:
remove console please